app/views/search/_granule_list.html.erb

```diff
@@ -97,7 +97,7 @@
               </div>
             </li>
           </ul>
-          <p class="busy panel-list-load-more" data-bind="visible: granules().isLoading()">
+          <p class="busy panel-list-load-more" data-bind="visible: granules.isLoading">
```

Comment:
Fix found!  It's the third permutation.  visible: granules().isLoading works - the spinner shows and disappears as it is supposed to.

While unrelated, it should be pointed out that the syntax does seem to be incorrect within index.html.erb.